frontend/src/components/Menu/Sidebar.spec.js

```diff
@@ -55,7 +51,12 @@ describe('Sidebar', () => {
       it('has first nav-item "navigation.profile" in navbar', () => {
         expect(wrapper.findAll('.nav-item').at(4).text()).toEqual('navigation.profile')
       })
-
+    })
+    // ----
+    describe('navigation Navbar (user has an elopage account)', () => {
+      it('has seven b-nav-item in the navbar', () => {
```

Comment:
eight?

Reply:
Das ist korrekt.
Das Wording wurde von vorheriger version des tests übernommen.
Aber es sind jeweils acht (user has elopage account) und sieben (user does not have elopage account) nav items.

Fixed with [a413851](https://github.com/gradido/gradido/pull/2099/commits/a41385141902b7611c7fbde289d68cc3d2fe4c6b).